x/auth/vesting/types/vesting_account_test.go

```diff
@@ -926,6 +926,83 @@ func TestClawback(t *testing.T) {
 	require.Equal(t, int64(50), stakeAmt.Int64())
 }
 
+func TestClawbackBeforeStartTime(t *testing.T) {
+	c := sdk.NewCoins
+	fee := func(x int64) sdk.Coin { return sdk.NewInt64Coin(feeDenom, x) }
+	stake := func(x int64) sdk.Coin { return sdk.NewInt64Coin(stakeDenom, x) }
```

Comment:
same with these. This doesn't add much, in fact, I think it makes it harder to read IMO.